src/eko/interpolation.py

```diff
@@ -174,7 +174,9 @@ def evaluate_Nx(N, logx, area_list):
             res += coef * (up - low) / (N + i)
     return res
 
-_atol_eps = 10*np.finfo(float).eps
+
+_atol_eps = 10 * np.finfo(float).eps
```

Comment:
This maybe should be lifted to a configuration, to be just a little more explicit than APFEL and not keep values hardcoded in the source. We're not on the same ground, this closely related to float precision and not an arbitrary number, but nevertheless having it in a configuration makes it even more explicit to the user.

Reply:
Oh yeah, than just raise a little bit its priority